src/lib.js

```diff
@@ -3267,93 +3230,56 @@ export async function getProfile(
 
       allSkyBlockProfiles.push(doc);
 
-      lastCachedSave = Math.max(lastCachedSave, doc.members[paramPlayer].last_save || 0);
+      lastCachedSave = Math.max(lastCachedSave, Date.now() || 0);
```

Comment:
This will always be `Date.now()`, right?

Reply:
Yeah, it's useless will remove tomorrow

Reply:
Solved, didn't remove found better way